src/main/java/net/mcreator/workspace/elements/ModElementManager.java

```diff
@@ -130,8 +130,6 @@ public boolean requiresElementGradleBuild(GeneratableElement generatableElement)
 		if (map.containsKey("triggersbuild") && map.get("triggersbuild").toString().equals("true"))
 			return true;
 
-		List<GeneratorTemplate> templates = generator.getModElementGeneratorTemplatesList(

```

Comment:
Why did you delete this?

Reply:
There is already a variable named `templates` before in the method, so it causes a build error.
![image](https://user-images.githubusercontent.com/38427877/129055088-5bbeadbd-9a39-4e94-a28e-cc45ccf0bedd.png)